components/avatar/skin.css

```diff
@@ -11,9 +11,9 @@ governing permissions and limitations under the License.
 */
 
 .spectrum-Avatar {
-  opacity: percent(var(--spectrum-avatar-small-opacity));
+  opacity: var(--spectrum-avatar-small-opacity);
```

Comment:
How does this exemplify in the output of `index-vars.css`? I've see some packages export with a fixed fallback `var(--spectrum-avatar-small-opacity, 0.3)` (seen in `@spectrum-css/actionbar`) and some with "system fallbacks" `var(--spectrum-avatar-small-opacity, --spectrum-global-disabled-opacity)` (seen in `@spectrum-css/avatar` ) , and am not sure what _should_ be expected in this area. Are there docs I could read on this subject?

Reply:
So we look at every instance of a variable, and we include a fallback -- either the variable referenced by the instance, or, if the instance does not reference another variable (i.e. an alias or global), the actual value.

This is (sort of) documented here: https://github.com/adobe/spectrum-css#variable-fallbacks

This allows us to reduce bundle sizes by not including every component's variable file while still allowing users to customize those variables (sort of documented at the end of this section https://github.com/adobe/spectrum-css#using-components).

Now, in the case of these percentage values, what we're going to see now is: 

```
.spectrum-Avatar {
  opacity: var(--spectrum-avatar-small-opacity, 1);
}
```

That is, if you override the opacity of an avatar with the following:

```
.spectrum-Avatar {
  --spectrum-avatar-small-opacity: 0.8;
}
```

The override will take effect. If that variable is not defined, you'll get the default of `1`.

Reply:
And the disabled version will output?
```
opacity: var(--spectrum-avatar-small-opacity, 0.3);
```

If so, great! Thanks for clarifying, I'll check out that link to see what else I can learn about the processing here.

Reply:
Yup, sure will! Thanks!

Reply:
@Westbrook ah, it shows up as:

```css
.spectrum-Avatar.is-disabled {
   opacity: var(--spectrum-avatar-small-opacity-disabled, 0.3);
}
```